### ISSUE TITLE
Sketchfab add-on on 4.0 beta only adds nulls to the scene

### ISSUE THREAD

**Issue by Loannl4** [sketchfab/blender-plugin#116]
In Blender 4.0  (currently in beta) on Windows 11, with the latest version of the addon (15), when a model is imported through the addon, the data is properly downloaded, but not imported. Only the nulls are added to the blender scene, with the proper hierarchy. When manually importing the model from the sketchfab download folder, everything works properly.

**Comment by toshicg** [comment]
Same thing happens with Blender 3.6 so I don't think it is because of Blender 4. I also saw a couple of similar reports. Looks like something to do with the API.

**Comment by emtea7** [comment]
Before, it happened very sporadically; now it happens (at least to me) even 90% of the time.

The add-on is effectively unusable.

**Comment by OutsideTheDome** [comment]
wtf is going on, why isn't the addon updated

**Comment by 4Bakers** [comment]
> Looks like something to do with the API.
Yup! I checked and the Python API had a few breaking changes for asset handling

**Comment by PadreZippo** [comment]
We shipped a new version (1.6.0) of the Blender plugin: https://github.com/sketchfab/blender-plugin/releases

This will fix the **_importer_**, as well as a couple other small bugs, but we have not yet updated the processing on our backend to handle Blender 4.0 .blend files, so the **_exporter_** feature still won't work yet.
